tests/unit/responsiveColumnsVisibility/Test.js

```diff
@@ -86,49 +87,52 @@ define([
 			rc.notifyCurrentValue("breakpoints");
 			responsiveColumnsVisibilityNode1 = document.getElementById("responsiveColumnsVisibilityAppdviewStack");
 		},
+		beforeEach: function () {
+			return when(new Promise(function (resolve) {
+				setTimeout(resolve, 20);
+			}));
+		},
 		"Desktop layout test initial views": function () {
 			this.timeout = 10000;
 
-			var appStartedDef = new Application(JSON.parse(stripComments(responsiveColumnsVisibilityconfig)),
-				responsiveColumnsVisibilityContainer1);
-			return appStartedDef.then(function (app) {
-				// we are ready to test
-				console.log("in responsiveColumnsVisibility tests app loaded. " + app.id);
-				rctestApp = app;
-
-				rcvsNode = document.getElementById("vs");
-				var testId = "rcaaa";
-				var testNode = document.getElementById(testId);
-				assert.isNotNull(testNode, testId + " must be here");
-				assert.isNotNull(rcvsNode, "rcvsNode must be here");
-				testLayout(leftLayout, '20%');
-				testLayout(centerLayout, 'fill');
-				testLayout(rightLayout, '20%');
-			});
+			return when(new Application(JSON.parse(stripComments(responsiveColumnsVisibilityconfig)),
+					responsiveColumnsVisibilityContainer1)
+				.then(function (app) {
+					// we are ready to test
+					console.log("in responsiveColumnsVisibility tests app loaded. " + app.id);
+					rctestApp = app;
+
+					rcvsNode = document.getElementById("vs");
+					var testId = "rcaaa";
+					var testNode = document.getElementById(testId);
+					assert.isNotNull(testNode, testId + " must be here");
+					assert.isNotNull(rcvsNode, "rcvsNode must be here");
+					testLayout(leftLayout, '20%');
+					testLayout(centerLayout, 'fill');
+					testLayout(rightLayout, '20%');
+				}));
```

Comment:
BTW, I noticed you changed the indentation across all of the test files.  I think the canonical way to write this would be:

```js
when(...).then(function () {
    // one level of indent
});
```
Not a big deal.